crates/astria-composer/src/block_relay/server.rs

```diff
@@ -0,0 +1,65 @@
+use std::net::SocketAddr;
+
+use color_eyre::eyre::{
+    self,
+    Context,
+};
+use proto::{
+    generated::block_relay::v1alpha1::top_of_block_relay_server::TopOfBlockRelayServer,
+    native::composer::block_relay::v1alpha1::bid::Bundle,
+};
+use tokio::sync::{
+    mpsc,
+    oneshot,
+};
+use tonic::transport::Server;
+
+pub(super) struct ProposerSubmission {
```

Comment:
Should probably just get rid of this and launch the server from `BlockRelay` - still a WIP.